tests/unit/builtin_monitors/kubernetes_monitor_test.py

```diff
@@ -77,6 +77,17 @@
 from six.moves import range
 
 
+class StringMatcher():
```

Comment:
Do we need to subclass this from `str`?

Reply:
In this case I would rather expect implementing an interface like Matcher in hamcrest in Java.
But with python's ducktyping approach, you just need a class implementing __eq__ :)